public/app/features/trails/TrailStore/TrailStore.ts

```diff
@@ -112,8 +149,8 @@ export class TrailStore {
   }
 
   load() {
-    this._recent = this._loadFromStorage(RECENT_TRAILS_KEY);
-    this._bookmarks = this._loadFromStorage(BOOKMARKED_TRAILS_KEY);
+    this._recent = this._loadRecentTrailsFromStorage();
+    this._bookmarkedStates = this._loadBookmarkedStatesFromStorage();
```

Comment:
not sure you need all the "States" suffixes everywhere (it feels unclear either way, what is a bookmark state vs just a bookmark?)